src/Integration/Connection/ConnectionWorkflow.cs

```diff
@@ -243,7 +253,7 @@ private SonarQubeOrganization AskUserToSelectOrganizationOnUIThread(IEnumerable<
 
             notifications.ProgressChanged(Strings.DownloadingServerSettingsProgessMessage);
 
-            var properties = await this.host.SonarQubeService.GetAllPropertiesAsync(token);
+            var properties = await this.host.SonarQubeService.GetAllPropertiesAsync(this.host.VisualStateManager.BoundProjectKey, token);
```

Comment:
After updating the client, I am now forced to pass a project key to this method.

Reply:
I've debugged what happens when binding for the first time. In that case, `this.host.VisualStateManager.BoundProjectKey` is `null`. However, this doesn't cause an exception since the web service call handles null in that case.
Technically, it's a bug, but it's not a new bug so we don't need worry about it now. I'll create a separate ticket for it.

FYI the impact isn't serious in any case. The code fetches the properties so it can get the regex to use for test projects - that's the only one it uses. The bug is that it will fetch the setting defined for the whole SQ instance, rather than for a specific SQ project. It just means that test projects in the solution might not be correctly identified if the user has set a different test regex for the SQ project being bound against.

Reply:
Update: the property is set correctly when re-opening a bound solution.